packages/calling-stateful-client/src/Converter.ts

```diff
@@ -182,7 +182,9 @@ export function convertSdkCallToDeclarativeCall(call: CallCommon): CallState {
     /* @conditional-compile-remove(hide-attendee-name) */
     hideAttendeeNames,
     /* @conditional-compile-remove(meeting-id) */
-    info: callInfo
+    info: callInfo,
+    /* @conditional-compile-remove(teams-meeting-conference) */
+    teamsMeetingConference: undefined
```

Comment:
Is this defaulting to undefined in the case that there is no conference coordinates? like in the case that this is a ACS call?

Reply:
Yes, meeting conference will contain phone information and it is available only in meeting scenario